scripts/gen_sqlc_docker.sh

```diff
@@ -24,5 +24,8 @@ echo "Fixing LEFT JOIN issue..."
 sed -i.bak -E 's/FamKeyFamily([[:space:]])+int32/FamKeyFamily\1sql.NullInt32/g' tarodb/sqlc/assets.sql.go
 sed -i.bak -E 's/FamKeyIndex([[:space:]])+int32/FamKeyIndex\1sql.NullInt32/g' tarodb/sqlc/assets.sql.go
 
+echo "Fixing DBTX issue..."
+sed -i.bak -E 's/WithTx\(tx \*sql\.Tx\) \*Queries/WithTx(tx DBTX) *Queries/g' tarodb/sqlc/db.go
```

Comment:
So this is a sort of hybrid approach where we now modify the _code_ itself? I thought we could either just use the main param, or use the interface?

Reply:
Hmm, I'm not sure what you mean. I did this so we didn't have to reference `*sql.Tx` directly. Unfortunately a function type isn't seen as compatible with another function type if the argument just takes a sub interface.
I tried a bunch of different things to get rid of the query creator function, but it all didn't work. It feels to me like the interface type restriction is a bit buggy still with 1.18.

Reply:
What I mean is that above we modify the auto generated code. Here, we're modifying actual Go code, but I'm unclear why this is required....

Is it the case then that this code actually won't compile (as is) if this script isn't run before normal compilation?